tsconfig.json

```
@@ -1,10 +1,12 @@
 {
   "compilerOptions": {
     "module": "commonjs",
```

Since we're doing a major bump it'd be nice to add a `"target": "es2017",` to match the cli
```suggestion
    "module": "commonjs",
    "target": "es2017",
```

Actually I just realised this is also necessary to guarantee the typed-error bump is safe

---

Agreed! Bumping to es6 is actually required for the new typed-error.